install/upgrade_to_2.13.php

```diff
@@ -37,10 +37,11 @@ class PluginFormcreatorUpgradeTo2_13 {
     */
    public function upgrade(Migration $migration) {
       $this->migration = $migration;
+      $this->migrateFkToUnsignedInt();
       $this->addFormAnswerTitle();
       $this->defaultValuesForTargets();
       $this->migrateItemtypeInQuestion();
-      $this->fixInconsistency();
+      // $this->fixInconsistency();
```

Comment:
Is this normal ?

Reply:
Yes : This method changes the type of some columns we also  change in this PR. Then it is useless. By the way I did not drop the method itself, then you can easily check its content.

Reply:
Method removed with a new commit